backends-common/postgres/src/main/java/org/apache/james/backends/postgres/PostgresConfiguration.java

```diff
@@ -40,6 +42,8 @@ public class PostgresConfiguration {
     public static final String NON_RLS_USERNAME = "database.non-rls.username";
     public static final String NON_RLS_PASSWORD = "database.non-rls.password";
     public static final String RLS_ENABLED = "row.level.security.enabled";
+    public static final String SSL_MODE = "ssl.mode";
+    public static final String SSL_MODE_DEFAULT_VALUE = "disable";
```

Comment:
Maybe `ALLOW` / `PREFER` would be better?

Support the SSL if needed, while still supporting no SSL in our test I guess.

Reply:
I think disabled by default is better, 
Similar to what r2dbc did

// I think because in the production environment, the database server and application should be run in a same private network, with no need SSL, less cost for encryption

Reply:
Ok, it makes sense.

Reply:
Actually makes sense. Will give it a shot thanks